code-challenges.test.js

```diff
@@ -17,35 +17,104 @@
+    lname = lname[0].toUpperCase() + lname.slice(1);
+    var fullname = fname + " " + lname;
+    return fullname + " is " + person.occupation;
+  });
+};
+
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false];
 // Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true];
 // Expected output: [ 2, 1, -1 ]
 
+describe("remainder", () => {
+  it("take in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", () => {
+    expect(remainder(hodgepodge1)).toEqual([2, 0, -1, 0]);
+    expect(remainder(hodgepodge2)).toEqual([2, 1, -1]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+// Create a function that takes in an a mixed data array
+// Use a higher order function .filter to filter through the returned array
+// Inside the method parameter declare the value as a placeholder
+// Go through the mixed data array by using typeof to get number values to placeholder
+// Use higher order function .map to go through the new array with only number values
+// Inside the method parameter declare the value modulo 3 to the placeholder to get the remainders of the numbers in the array.
+
+const remainder = (array) => {
+return array.filter(value => typeof value === 'number').map(value => value % 3)
+}
+
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
+const cubeAndSum1 = [2, 3, 4];
 // Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
+const cubeAndSum2 = [0, 5, 10];
 // Expected output: 1125
 
+describe("adder", () => {
+  it("take in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+    expect(adder(cubeAndSum1)).toEqual(99);
+    expect(adder(cubeAndSum2)).toEqual(1125);
+  });
+});
 
 // b) Create the function that makes the test pass.
+
+// Pseudocode:
+//Create a function that takes in an array of numbers
+// Use the higher order function .map to go through the returned array
+// In the method use the built in method .pow to iterate on all of the numers with the exponent three times passed to placeholder
+// Use the built in method .reduce to return a single value out of the sum of everything inside the arrays respectively
+
+const adder = (array) => {
+  return array.map((value) => Math.pow(value, 3)).reduce((previousValue, currentValue) => previousValue + currentValue, 0);
+};
```

Comment:
Indentation, testing, pseudocode, and function look great. Please make sure you are documenting the test results before and after creating a function.